### ISSUE TITLE
validation loss dont decrease under 0.45

### ISSUE THREAD

**Issue by johnSmith1990** [princewang1994/TextSnake.pytorch#50]
Hi dear,

thank you for your code.
but my both train and validation loss don't decrease under 0.45 with 400 epochs. May you help me for more config such as learning rate and so on?

regards

**Comment by princewang1994** [comment]
What dataset you are training on? if total-text or icdar2013/15, the configuration in README can reproduce the result in table. If not in two datasets above, please check dataloader you implement is correct first.
